### PR TITLE
Katello 3.12 Docs

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@jturel here is the pr so far

File: plugins/katello/3.12/release_notes/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+layout: plugins/katello/documentation
+title: Release Notes
+version: 3.12
+---
+
+# Katello Nightly () Release Notes
```
Comment (jturel) [inline plugins/katello/3.12/release_notes/release_notes.md]: Can you change this to (here and also in nightly release notes so we won't have to manually change it later)

```
# Katello {{ page.version }} Release Notes
```

File: plugins/katello/3.12/installation/index.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
+---
+layout: plugins/katello/documentation
+title: Katello Installation
+version: 3.12
+foreman_version: 1.22
+latest: nightly
```
Comment (jturel) [inline plugins/katello/3.12/installation/index.md]: 'latest' should be 3.12 here so the banner displays properly

File: plugins/katello/3.12/release_notes/release_notes.md (hunk 1 above)
Comment (jturel) [reply plugins/katello/3.12/release_notes/release_notes.md]: Also, by looking at the CHANGELOG you created on KATELLO-3.12 branch you should be able to whip up some release notes. Let me know if you need help with that

**Comment by jturel** [comment]
This is looking good. One thing I discovered is that the version needs to be in quotes. You can fix yours like this from within plugins/katello/3.12

```
find . -type f -print0 | xargs -0 sed -i "s/version: 3.12/version: '3.12'/g"
```

Without this, when 3.13 is released, the 3.12 docs will select the wrong version in the dropdown (3.13 instead of 3.12)

**Comment by jturel** [comment]
Nice! One other thing that should be included is updated API docs per https://github.com/theforeman/tool_belt/blob/master/procedures/katello/branch.md.erb

If you want we can handle as a separate PR but I don't think there's harm including it all together here

**Comment by jturel** [comment]
API docs aren't showing up it looks like the /api/ dir isn't quite organized properly. Check this out:

```
[jturel@strangeways katello]$ ll 3.11/api/
total 452
drwxrwxr-x. 5 jturel jturel   4096 May  3 13:27 apidoc
-rw-rw-r--. 1 jturel jturel 226015 May  3 13:27 apidoc.en.html
-rw-rw-r--. 1 jturel jturel 223321 May  3 13:27 apidoc.html
-rw-rw-r--. 1 jturel jturel    521 May  6 15:14 index.md
[jturel@strangeways katello]$ ll 3.12/api/                                                                                                                                  
total 16
drwxrwxr-x. 4 jturel jturel 4096 May  6 15:14 apidoc
-rw-rw-r--. 1 jturel jturel  890 May  6 15:14 apidoc.html
drwxrwxr-x. 3 jturel jturel 4096 May  6 15:14 apipie-cache
-rw-rw-r--. 1 jturel jturel  521 May  6 15:14 index.md

...

[jturel@strangeways katello]$ ll 3.12/api/apidoc
total 8
drwxrwxr-x. 3 jturel jturel 4096 May  6 15:14 javascripts
drwxrwxr-x. 3 jturel jturel 4096 May  6 15:14 stylesheets
[jturel@strangeways katello]$ ll 3.11/api/apidoc
total 472
drwxrwxr-x.   3 jturel jturel   4096 May  3 13:27 javascripts
drwxrwxr-x.   3 jturel jturel   4096 May  3 13:27 stylesheets
drwxrwxr-x. 112 jturel jturel  20480 May  3 13:27 v2
-rw-rw-r--.   1 jturel jturel 226921 May  3 13:27 v2.en.html
-rw-rw-r--.   1 jturel jturel 224227 May  3 13:27 v2.html

```

I think some things need to be rearraged to match 3.11 and other versions. Make sure to test locally with:

```
docker run -it --rm -v `pwd`:/srv/jekyll:Z -p 4000:4000 jekyll/jekyll jekyll serve --watch
```

**Comment by chris1984** [comment]
@jturel fixed up the api docs, dir looks the same as the 3.11 now and when I load up the site with rake server the 3.12 api docs look like they should. Let me if it works for you

**Comment by jturel** [comment]
Looks great! Thanks for your diligence.

**Review by jturel — APPROVED** [review]
ACK!

**Comment by jturel** [comment]
This needs to be merged in conjunction (or after) https://github.com/theforeman/theforeman.org/pull/1367

**Pull request merged**